Objects/typeobject.c

```diff
@@ -3581,6 +3581,8 @@ type_new_alloc(type_new_ctx *ctx)
     et->ht_module = NULL;
     et->_ht_tpname = NULL;
 
+    _PyObject_SetDeferredRefcount((PyObject *)et);
```

Comment:
Do we want to set this for static types via `PyType_Ready` as well? Or I forgot if they're already immortal (I suspect they are).

Reply:
They are already immortal